frontend/src/components/navbar/NavBar.cy.js

```diff
@@ -0,0 +1,32 @@
+import Navbar from "./NavBar"
+
+describe('Navbar', () => {
+  it('NavBar should displays 2 links with nav-item class', () => {
+    cy.mount(<Navbar />)
+
+    cy.contains('Farcebook') // checks if the navbar has the correct title
```

Comment:
I think you may have moved Farcebook out after this test as it now fails. Could you please comment this line out for now.